apps/dialtone-documentation/docs/components/empty-states.md

```diff
@@ -0,0 +1,378 @@
+---
+title: Empty state
+description: When there's no data to display in a list, table, or other information container, we use an ‘Empty State’ component to guide users, clarifying the expected information and suggesting ways to populate it.
+status: ready
+thumb: true
+image: assets/images/components/empty-state.png
+figma: planned
+storybook: https://dialtone.dialpad.com/vue/?path=/story/components-card--default
+---
+
+<code-well-header>
+  <dt-empty-state
+    illustration-name="mind"
+    header-text="Nothing to see here"
+    body-text="Looks like there's no data to display here."
+  >
+    <template #body>
+      <div class="d-mt8 d-stack d-stack--row d-stack--gap-300">
+        <button class="d-btn">Click me</button>
+        <button class="d-btn d-btn--primary">Click me</button>
+      </div>
+    </template>
+  </dt-empty-state>
+</code-well-header>
+
+## Morphology & Anatomy
+
+Image with bullets
+<code-well-header>
+<img class="d-w90p" alt="empty state bullets" src="/assets/images/components/empty-state01.png">
```

Comment:
This is really small. I can't read the numbers.

Reply:
Image updated